homeassistant/helpers/area_registry.py

```diff
@@ -132,7 +157,13 @@ async def async_load(self) -> None:
 
         if data is not None:
             for area in data["areas"]:
-                areas[area["id"]] = AreaEntry(name=area["name"], id=area["id"])
+                normalized_name = area.get(
```

Comment:
Let’s compute it on load so we can update the normalization in the future

Reply:
👍  good call.

Done